### PR TITLE
Milestone: Desktop Verson

### PR DESCRIPTION
**Pull request by Gleennkar**
## Created the Desktop Version of My Portfolio
- Added an index.html to the desktop version 
- Added style.css to the Desktop version
- check for linter errors
- Used Figma template 1 as reference

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -99,16 +112,23 @@ <h2>Multi-Post <br> Stories</h2>
...
 
         <div class="card">
-            <img src="./Images/SnapshootPortfolio2.png" alt="SnapshootPortfolio2">
+            <img src="./Images/SnapshootPortfolio2.png" alt="SnapshootPortfolio2" class="img1">
+            <img src="./Images/SnapshootPortfolio-desktop3.png" alt="SnapshootPortfolio-desktop3" class="img2">
+            <img src="./Images/SnapshootPortfolio2.png" alt="SnapshootPortfolio2" class="img3">
 
-            <h2>Multi-Post <br>Stories</h2>
+            <div>
+               <h2>Multi-Post <br>Stories</h2>
```
Comment (ishimwezachee) [inline index.html:131]: - [ ]  You've done a good job, your project look nice, but its missing a very important feature, the responsiveness for screens that are above something around 1120px width,is not ok, the cards are getting fixed. 

Your design | Figma design
--- | ---
image) | image

File: style.css [hunk 2]
```diff
@@ -378,3 +380,224 @@ button:active {
...
+    width: 100vw;
+  }
+
+  .footer-wrap {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+}
```
Comment (ishimwezachee) [inline style.css:398]: - [ ]  You should use media queries for the Desktop breakpoint in the CSS file. (the value for min-width property should be in range of 768px or wider)

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @Gleennkar , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Linters are passing 👍 
PR is well described 👍 
Project is running successfully 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @Gleennkar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**